README.adoc

```diff
@@ -1,12 +1,23 @@
-# Stackable Agent
+= Stackable Agent
 
-## Purpose
-This agent is the component in the Stackable stack which is run locally on all nodes to enact changes that are necessary to deploy and manager the software components that are managed by Stackable.
+== Purpose
+The purpose of the Stackable architecture, if I had to put it in one sentence is to "run Kubernetes workloads without containers".
```

Comment:
I think I disagree with that as it's our plan to also run containers and this readme should be about the agent and not the whole architecture. At least the beginning.

Why not write something along these lines: "The Stackable Agent is an alternative to the Kubernetes Kubelet that executes Pods not in containers but using systemd as its backend. It is implemented in Rust as a Krustlet provider." (+ link to krustlet)

Reply:
I've tried again, if this still doesn't fit the bill I'd suggest that we get together with a gdoc and collaborate directly on a final version.